src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -104,6 +110,26 @@ public ColibriConferenceImpl(XmppConnection connection)
     }
 
     /**
+     * Checks if this instance has been disposed already and if so prints
+     * a warning message.
+     *
+     * @param operationName the name of the operation that will not happen and
+     * should be mentioned in the warning message.
+     *
+     * @return <tt>true</tt> if this instance has not been disposed yet or
+     * <tt>false</tt> otherwise.
+     */
+    private boolean assertNotDisposed(String operationName)
```

Comment:
I don't like the method name assertNotDisposed because:

1. It sets the wrong expectation for me. I expect assert methods to throw an exception (because that's how I've seen them behave elsewhere like in JUnit). Apple's Dictionary.app says that assert means "state a fact or belief confidently and forcefully". I think "confidently and forcefully" aligns with "throw an exception".

2. Its uses read unnecessarily non-trivially. You're only ever using the method in the form "if (!assertNotDisposed(...)) return ..." which a double negative. I believe the source code will be easier to read without a double negative.